evergreen.yml

```diff
@@ -1510,26 +1496,14 @@ tasks:
         variant: "*"
       - name: rustfmt
         variant: "*"
-      - name: windows-unit-test
-        variant: "windows-64"
-      - name: windows-integration-test
-        variant: "windows-64"
-      - name: windows-result-set-test
-        variant: "windows-64"
-      - name: ubuntu-unit-test
-        variant: "ubuntu2204"
-      - name: ubuntu-integration-test
-        variant: "ubuntu2204"
-      - name: ubuntu-result-set-test
-        variant: "ubuntu2204"
-      - name: asan-unit-test
+      - name: unit-test
         variant: "*"
-      - name: asan-integration-test
+      - name: integration-test
         variant: "*"
-      - name: windows-sign
```

Comment:
Side note: Is there a reason that the `asan-unit-test` and `asan-integration-test` needed to use all variants?

Reply:
That may have been a small oversight since there's only one variant that runs the asan tests.